### PR TITLE
Added valid Slack invite link

### PR DESCRIPTION
**Pull request by RafeyIqbalRahman**
### Summary

I have added the valid Slack invite link and removed the invalid one.

### Details and comments

The valid link, i.e., https://qiskit.slack.com/join/shared_invite/zt-fybmq791-hYRopcSH6YetxycNPXgv~A#/, has been added in README.md. This PR target this open issue: https://github.com/Qiskit/qiskit/issues/976

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ and if you want to run your experiments on real systems you need the 5th element
...
 
-We use [GitHub issues](https://github.com/Qiskit/qiskit/issues) for tracking requests and bugs. Please use our [slack](https://join.slack.com/t/qiskit/shared_invite/enQtODQ2NTIyOTgwMTQ3LTI0NzM2NzkzZjJhNDgzZjY5MTQzNDY3MGNiZGQzNTNkZTE4Nzg1MjMwMmFjY2UwZTgyNDlmYWQwYmZjMjE1ZTM) for discussion and simple questions. To join our Slack community use the [link](https://qiskit.slack.com/join/shared_invite/zt-fybmq791-hYRopcSH6YetxycNPXgv~A#/). For questions that are more suited for a forum we use the Qiskit tag in the [Stack Exchange](https://quantumcomputing.stackexchange.com/questions/tagged/qiskit).
+We use [GitHub issues](https://github.com/Qiskit/qiskit/issues) for tracking requests and bugs. Please use our [slack](https://qiskit.slack.com/join/shared_invite/zt-fybmq791-hYRopcSH6YetxycNPXgv~A#/) for discussion and simple questions. To join our Slack community use the [link](https://qiskit.slack.com/join/shared_invite/zt-fybmq791-hYRopcSH6YetxycNPXgv~A#/). For questions that are more suited for a forum we use the Qiskit tag in the [Stack Exchange](https://quantumcomputing.stackexchange.com/questions/tagged/qiskit).
```
Comment (gunchamalik) [inline README.md]: change looks fine.

**Comment by gunchamalik** [comment]
@RafeyIqbalRahman I was reviewing the changes and was about to approve when I realized that I am not seeing any issues with the link that is currently provided on the Readme.md. I have tried the links on the below links:

Element |  Readme.md
-- | --
qiskit-terra | https://github.com/Qiskit/qiskit-terra
qiskit-aer | https://github.com/Qiskit/qiskit-aer
qiskit-ignis | https://github.com/Qiskit/qiskit-ignis
qiskit-aqua | https://github.com/Qiskit/qiskit-aqua
Docs or Qiskit Meta-package | https://github.com/Qiskit/qiskit

So, I am wondering if this change is really required. Please let me know what error are you seeing when you click on those links.

**Comment by RafeyIqbalRahman** [comment]
> @RafeyIqbalRahman I was reviewing the changes and was about to approve when I realized that I am not seeing any issues with the link that is currently provided on the Readme.md. I have tried the links on the below links:
> 
> Element	Readme.md
> qiskit-terra	https://github.com/Qiskit/qiskit-terra
> qiskit-aer	https://github.com/Qiskit/qiskit-aer
> qiskit-ignis	https://github.com/Qiskit/qiskit-ignis
> qiskit-aqua	https://github.com/Qiskit/qiskit-aqua
> Docs or Qiskit Meta-package	https://github.com/Qiskit/qiskit
> So, I am wondering if this change is really required. Please let me know what error are you seeing when you click on those links.

This is what I got when I opened the Slack link provided on Docs or Qiskit Meta-package	https://github.com/Qiskit/qiskit. Other repos have the same issue.

issue

**Comment by RafeyIqbalRahman** [comment]
You won't face this issue in qiskit-terra https://github.com/Qiskit/qiskit-terra since my PR there has been merged.

merge

**Comment by gunchamalik** [comment]
@RafeyIqbalRahman thx for confirming. I see the issue and the issue is resolved with this fix.

**Review by gunchamalik — APPROVED** [review]
fix resolves the issue.

**Review by gunchamalik — APPROVED** [review]
fix resolves the issue.

**Review by gunchamalik — APPROVED** [review]
fix resolves the issue

**Comment by RafeyIqbalRahman** [comment]
You're welcome. :)

**Review by gunchamalik — DISMISSED** [review]
attempting again to confirm my review.

**Comment by SooluThomas** [comment]
The links (in qiskit.org) is working now. Please correct me if I'm wrong.

**Comment by RafeyIqbalRahman** [comment]
Yes, @SooluThomas, the link on qiskit.org is valid. While in the README.md file, it's invalid.

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ and if you want to run your experiments on real systems you need the 5th element
...
-We use [GitHub issues](https://github.com/Qiskit/qiskit/issues) for tracking requests and bugs. Please use our [slack](https://join.slack.com/t/qiskit/shared_invite/enQtODQ2NTIyOTgwMTQ3LTI0NzM2NzkzZjJhNDgzZjY5MTQzNDY3MGNiZGQzNTNkZTE4Nzg1MjMwMmFjY2UwZTgyNDlmYWQwYmZjMjE1ZTM) for discussion and simple questions. To join our Slack community use the [link](https://join.slack.com/t/qiskit/shared_invite/enQtODQ2NTIyOTgwMTQ3LTI0NzM2NzkzZjJhNDgzZjY5MTQzNDY3MGNiZGQzNTNkZTE4Nzg1MjMwMmFjY2UwZTgyNDlmYWQwYmZjMjE1ZTM). For questions that are more suited for a forum we use the Qiskit tag in the [Stack Exchange](https://quantumcomputing.stackexchange.com/questions/tagged/qiskit).
+We use [GitHub issues](https://github.com/Qiskit/qiskit/issues) for tracking requests and bugs. Please use our [slack](https://qiskit.slack.com/join/shared_invite/zt-fybmq791-hYRopcSH6YetxycNPXgv~A#/) for discussion and simple questions. To join our Slack community use the [link](https://qiskit.slack.com/join/shared_invite/zt-fybmq791-hYRopcSH6YetxycNPXgv~A#/). For questions that are more suited for a forum we use the Qiskit tag in the [Stack Exchange](https://quantumcomputing.stackexchange.com/questions/tagged/qiskit).
```
Comment (SooluThomas) [inline README.md]: I would suggest using this link - http://ibm.co/joinqiskitslack instead. 

```suggestion
We use [GitHub issues](https://github.com/Qiskit/qiskit/issues) for tracking requests and bugs. Please use our [slack](http://ibm.co/joinqiskitslack) for discussion and simple questions. For questions that are more suited for a forum we use the Qiskit tag in the [Stack Exchange](https://quantumcomputing.stackexchange.com/questions/tagged/qiskit).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -24,6 +24,10 @@ Changed
 -------
 
 - Increased the `qiskit-ibmq-provider` version to the latest release 0.3.3.
+- Added valid Slack invite link.
+- Added direct link for navigating to "My Account" on IBM Quantum Experience.
+- Removed broken image links for My Account screen and API Token access.
+- Added a visible image for API token access.
```
Comment (SooluThomas) [inline CHANGELOG.md]: This need not be added.
```suggestion
```

File: docs/install.rst [hunk 4]
```diff
@@ -124,16 +124,15 @@ To configure your account, you create a local configuration file which includes
...
-
-.. image:: /images/figures/install_0.png
-   :alt: Image of where to find the section 'My accounts'.
+**2.**  Navigate to https://quantum-computing.ibm.com/account to view your account settings.
 
 **3.** Click on **Copy token** to copy the token to your clipboard.
 Temporarily paste this API token into your favorite text editor so you can use it later to create
 an account configuration file.
 
-.. image:: /images/figures/install_1.png
```
Comment (SooluThomas) [inline docs/install.rst]: I don't think that the link to the image is broken? (https://qiskit.org/documentation/install.html#access-ibm-quantum-systems)

File: docs/install.rst [hunk 5]
```diff
@@ -124,16 +124,15 @@ To configure your account, you create a local configuration file which includes
 
 **1.** `Create a free IBM Quantum Experience account <https://quantum-computing.ibm.com/login>`__.
 
-**2.**  Navigate to **My Account** to view your account settings.
-
-.. image:: /images/figures/install_0.png
-   :alt: Image of where to find the section 'My accounts'.
+**2.**  Navigate to https://quantum-computing.ibm.com/account to view your account settings.
```
Comment (SooluThomas) [inline docs/install.rst]: I would keep the image as it is and simply add a link to `My Account`
```suggestion
**2.**  Navigate to `My Account <https://quantum-computing.ibm.com/account>`__ to view your account settings.

.. image:: /images/figures/install_0.png
    :alt: Image of where to find the section 'My accounts'.
```

**Review by SooluThomas — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (RafeyIqbalRahman) [reply CHANGELOG.md]: > This need not be added.

I'll remove this.

File: README.md (hunk 2 above)
Comment (RafeyIqbalRahman) [reply README.md]: > I would suggest using this link - http://ibm.co/joinqiskitslack instead.

Good suggestion.

File: docs/install.rst (hunk 4 above)
Comment (RafeyIqbalRahman) [reply docs/install.rst:136]: > I don't think that the link to the image is broken? (https://qiskit.org/documentation/install.html#access-ibm-quantum-systems)

You're right. I don't know for whatever reason I was getting a 404 error. Nevermind.

Comment (RafeyIqbalRahman) [reply docs/install.rst]: > I don't think that the link to the image is broken? (https://qiskit.org/documentation/install.html#access-ibm-quantum-systems)

The link you've quoted is from the Documentation page. However, I'm referring to "broken" is this:

Comment (SooluThomas) [reply docs/install.rst]: The docs are inside the `docs/` directory. You should be looking for the image inside `docs/` https://github.com/Qiskit/qiskit/blob/master/docs/images/figures/install_1.png

File: CHANGELOG.md [hunk 6]
```diff
@@ -23,8 +23,6 @@ Changelog](http://keepachangelog.com/en/1.0.0/).
 Changed
 -------
 
-- Increased the `qiskit-ibmq-provider` version to the latest release 0.3.3.
-
```
Comment (SooluThomas) [inline CHANGELOG.md]: I'm guessing that this was deleted by mistake?

File: docs/install.rst (hunk 4 above)
Comment (RafeyIqbalRahman) [reply docs/install.rst]: Got it. Let me undo this.

File: CHANGELOG.md (hunk 6 above)
Comment (RafeyIqbalRahman) [reply CHANGELOG.md]: But you said that there was no need for this.

Comment (SooluThomas) [reply CHANGELOG.md]: This was already in there. I asked you to just remove anything that you added to the CHANGELOG.md. Sorry for being unclear.

Comment (RafeyIqbalRahman) [reply CHANGELOG.md]: Got it. This has been added now.

File: CHANGELOG.md [hunk 7]
```diff
@@ -22,10 +22,8 @@ Changelog](http://keepachangelog.com/en/1.0.0/).
 
 Changed
 -------
-
 - Increased the `qiskit-ibmq-provider` version to the latest release 0.3.3.
```
Comment (SooluThomas) [inline CHANGELOG.md]:
```suggestion

- Increased the `qiskit-ibmq-provider` version to the latest release 0.3.3.

```

**Review by SooluThomas — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**